examples/wesleycsj-helloWorld.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *	Author: WesleyCSJ
+ *  Description: Render four colors on screen with 2s delay
+ *
+ *
+ */
+#include <iostream>
```

Comment:
Since you are not using cout/cerr/cin I think you should delete that unnecessary include

Reply:
Fine.